full-service/src/json_rpc/wallet.rs

```diff
@@ -514,6 +539,28 @@ where
                 account_map,
             }
         }
+        JsonCommandRequest::get_all_view_only_accounts => {
+            // TODO(cc) how to refactor this so that view_only and regular accounts can
```

Comment:
I think that since we're only doing this twice, that we don't need to worry so much about refactoring. If we were doing this multiple times then we should have a helper function. Let's remove the TODO comment.